### ISSUE TITLE
preview-docs command does not read entrypoint from config file

### ISSUE THREAD

**Issue by Blacklands** [Redocly/openapi-cli#166]
Since version `1.0.0-beta.1`, it seems that the `preview-docs` command does no longer attempt to check the configuration file (`.redocly.yaml`) for where the specification is located.
That means that the `entrypoint` parameter for the command is now required, even if the configuration file exists and has a path to the specification defined in the `apiDefinitions` sections.

In the older versions (`0.x.x`), the command used to check the config file and use the path from there if it exists, so specifying the `entrypoint` parameter was then not needed.
It's also now inconsistent with the other commands (`lint` and `bundle`), since these _do_ use the path from the config file.

**Comment by adamaltman** [comment]
FYI: `preview-docs` was semi-broken before `1.0.0-beta3`. 

Can you provide how to reproduce the issue?  Can you share a snippet of your `.redocly.yaml` file so we can see the `apiDefinitions` section?

**Comment by RomanHotsiy** [comment]
@Blacklands could you verify this is still an issue with the beta?

**Comment by Blacklands** [comment]
I should have clarified which version I am using exactly, sorry. I was already using the latest beta (`1.0.0-beta.3`) at this point. I started with `1.0.0-beta.1`, but then realized that `preview-docs` was fairly broken, yes. I just wanted to point out that this problem has existed since the first beta version, but it's still there now.

I think I also did a bad job explaining the problem: It's not that the command somehow _can't_ read from the config file, it's that you simply can't leave out the `entrypoint` parameter when you use the command.
From how it looks like, the `entrypoint` parameter has been marked as "required" in the code. If you try to leave it out and just call `openapi preview-docs`, you get
```
Not enough non-option arguments: got 0, need at least 1
```
If you call `openapi preview-docs help`, it also says
```
cli.js preview-docs <entrypoint>

Preview API reference docs for the specified definition.

Positionals:
  entrypoint                                                 [string] [required]

Options:
[...]
```
The other two commands (`bundle` and `lint`) say this instead:
```
cli.js bundle [entrypoints...]

Bundle definition.

Positionals:
  entrypoints                                              [array] [default: []]
```

So all that might be missing here is making the `entrypoint` parameter for `preview-docs` optional/adding a default value.

**Comment by adamaltman** [comment]
@Blacklands preview-docs is quite different from lint or bundle. Lint or bundle can operation on entrypoint**s** (plural). Preview-docs can only operate on one and only one entrypoint at a time. Which entrypoint from `apiDefinitions` would you expect it to use (assuming you have more than one in `apiDefinitions`?

**Comment by adamaltman** [comment]
Also, it does check the config file, so you can do this (where `aliasFromApiDefinitions` is set in the `apiDefinitions` section:

`openapi preview-docs aliasFromApiDefinitions`

Also, the `referenceDocs` section is utilized too.

**Comment by RomanHotsiy** [comment]
@adamaltman I believe the old openapi-cli used the first definition from .redocly.yaml if none was supplied via CLI args. I think we can fix that.

**Comment by Blacklands** [comment]
Yes, that's what I meant when I said that I didn't express it very well (again, sorry). It does use the config file, it just doesn't allow you to leave out the `entrypoint` parameter and have it use the spec that's linked in the `apiDefinitions` section automatically.
Like @RomanHotsiy said, it used to do that (picking the first definition from that section) in the old version. I was using that, and was surprised when it suddenly didn't work anymore in the beta.
It's not a huge issue, but it makes things easier (if you only have one spec at a time, like I have).

**Comment by RomanHotsiy** [comment]
It will land in the upcoming release

**Comment by RomanHotsiy** [comment]
@Blacklands just fyi: landed in 1.0.0-beta.4
